libs/blocks/caas-config/caas-config.js

```diff
@@ -297,6 +319,12 @@ const UiPanel = () => html`
     prop="loadMoreBtnStyle"
     options=${defaultOptions.loadMoreBtnStyle}
   />
+  <${Input} label="Custom Card HTML" prop="customCard" type="text" />
```

Comment:
Not sure how much this is used - should it be a textArea so that authors can see?  If it's seldom used I think it's fine as is.

Reply:
I was thinking about this and I think text is fine for now. If a use case appears that requires longer, then I can update it after the fact.